install-guides/ubuntu20.md

```diff
@@ -0,0 +1,201 @@
+# Deploying Pixelfed on Ubuntu 20.04.4 LTS (Focal Fossa)
+Guide by @shlee@aus.social - Version 0.1
+
+## NOTES
+* PHP 7.4
+
+## TODO
+* Update redis and mariadb to use unixsockets over tcp. (Investigate if this is worth it)
+
+## Part 0 - Setup the VM and update the DNS for A and AAAA records
+![image](https://user-images.githubusercontent.com/17537000/168226273-9b89cc51-11ca-4ace-9137-99f2401b3b28.png)
+![image](https://user-images.githubusercontent.com/17537000/168226310-e5c4f24a-f93c-4234-8e23-e2ce865ca988.png)
+
+## Part 1 - Upgrade Ubuntu 20.04 LTS
+```
+apt update
+apt upgrade -y
+reboot now
+```
+
+## Part 2 - Redis - Install
+```
+apt -y install redis-server
+systemctl enable redis-server
+```
+
+## Part 3 - MariaDB - Install
+```
+apt -y install mariadb-server
+systemctl enable mariadb
+mysql_secure_installation
+mysql -u root -p
+```
+
+Run the SQL query to create the pixelfed DB
+```
+create database pixelfed;
+grant all privileges on pixelfed.* to 'pixelfed'@'localhost' identified by 'secretpasswordhere';
+flush privileges;
+exit;
+```
+
+## Part 4 - Setup dependent packets
+```
+apt -y install ffmpeg 
+apt -y install jpegoptim optipng pngquant gifsicle 
+apt -y install unzip zip
+```
+
+## Part 5 - PHP - Install
+```
+apt -y install php7.4-fpm php7.4-cli
+### Install additional PHP modules not installed by default
+apt -y install php7.4-bcmath php7.4-curl php7.4-gd php7.4-intl php7.4-mbstring php7.4-redis php7.4-xml php7.4-zip php7.4-mysql
+```
+
+### PHP - Setup
+```
+nano /etc/php/7.4/fpm/php.ini
+```
+Edit these lines
+```
+    post_max_size (default 8M, set this around or slightly greater than your desired post size limit)
```

Comment:
i think this might need to be raised when/if importing is added, but right now the recommendation is fine